### ISSUE TITLE
 System.IO.FileNotFoundException after install via nuget

### ISSUE THREAD

**Issue by Indigo744** [punker76/gong-wpf-dragdrop#318]
Exact same issue as #240 

I happen to have the exact same issue... And I'm banging my head at it since several hours, without a shred of success... 😞 

## Error message

Basically, I got same message as @wowfood but in French:  `Could not load file or assembly '...' or one of its dependencies. The system cannot find the file specified.`

The message differs between version on the `PublicKeyToken`.

**With version v1.1.0**
`
System.Windows.Markup.XamlParseException : FileNotFoundException : 'Impossible de charger le fichier ou l'assembly 'GongSolutions.WPF.DragDrop, PublicKeyToken=null' ou une de ses dépendances. Le fichier spécifié est introuvable.'
`

**With latest version v2.0.1**
`
System.Windows.Markup.XamlParseException : FileNotFoundException : 'Impossible de charger le fichier ou l'assembly 'GongSolutions.WPF.DragDrop, PublicKeyToken=91f1945125b7a587' ou une de ses dépendances. Le fichier spécifié est introuvable.'
`

## Environment
 - Windows 10 Pro 1809
 - Visual Studio 2017
 - Project : .NET Framework 4.5

## XAML code sample

Don't know if it's relevant, but anyway.

I only followed the first part of the usage wiki page by adding `dd:DragDrop.*` to my `ItemsControl`.

```xaml
<!--  Widgets  -->
<ItemsControl
	dd:DragDrop.IsDragSource="True"
	dd:DragDrop.IsDropTarget="True"
	ItemsSource="{Binding Path=Widgets}">
	<ItemsControl.ItemsPanel>
		<ItemsPanelTemplate>
			<WrapPanel IsItemsHost="True" />
		</ItemsPanelTemplate>
	</ItemsControl.ItemsPanel>
</ItemsControl>
```

## What I tried

 - Full cleanup of solution, full re-regenerate
 - Deleting `.suo` solution file 
 - Deleting `packages` solution folder
 - Trying version 1.1.0 and 2.0.1
 - Trying different xmlns (`xmlns:dd="urn:gong-wpf-dragdrop"` and `xmlns:dd="clr-namespace:GongSolutions.Wpf.DragDrop;assembly=GongSolutions.Wpf.DragDrop"` )

I'm out of idea... I need your help.

@wowfood can you be more precise with the "_and all the other stuff_"?

Thanks a lot.

**Comment by wowfood** [comment]
Sorry I can't be much more precise as this was a long while back, and we dropped drag and drop as a requirement.

My "something isn't right" cleanup process though, is as follows

Clear VS cache.
Close VS
Delete all directories (except the .git)
git reset --hard

Or as I like to call the process, nuking it from orbit.  But this won't work if you don't have things stored in git.

**Comment by Indigo744** [comment]
Thanks for getting back 😄 

After fiddling for a while, I managed to get it to work.

I followed the first part of [this article](https://www.matteopozzani.com/visual-studio-cache-cleanup) to clean the VS cache. No luck, same issue 😞 . Then I tried the full reset. Same issue 😞 . Then I tried the nuking from orbit as you say (we use TFSVC but process is similar), and.... same issue with a clean repo 😞 

Then I remembered that the project using WpfDragDrop is actually a library. The main application is in another project and which referenced the project library.

On a hunch, I added the GongSolutions.Wpf.DragDrop Nugget to the main project, compiled and run and... It works! 😄 

So the main takeaway here is: **if a library needs a Nugget, the main project needs it as well**. Now I'll know...

Thanks a lot!
